pkg/scheduler/framework/handle/handle.go

```diff
@@ -84,4 +85,8 @@ type UnitFrameworkHandle interface {
 	FindStore(commonstore.StoreName) commonstore.Store
 
 	GetMaxWaitingDeletionDuration() time.Duration
+
+	// TODO: cleanup the following function methods.
```

Comment:
Move theis line to #L87 considering that the GetMaxWaitingDeletionDuration should also be cleand up

Reply:
fixed